pkg/domain/config.go

```diff
@@ -0,0 +1,40 @@
+package domain
```

Comment:
This should probably go in the same package as the db implementation, similar to https://github.com/asecurityteam/awsconfig-filterd/blob/master/pkg/filter/resource.go

Reply:
OK. I wasn't sure about this one